app/cdap/components/GridTable/index.tsx

```diff
@@ -289,10 +349,21 @@ export default function GridTable() {
           </TableBody>
         </Table>
       )}
+      {openDirectivePanel && (
+        <DirectiveInput
+          columnNamesList={headersNamesList}
+          onDirectiveInputHandler={(directive) => {
+            addDirectives(directive);
+            setDirectivePanel(false);
+          }}
+          onClose={() => setDirectivePanel(false)}
+          openDirectivePanel={openDirectivePanel}
+        />
+      )}
       {loading && (
-        <div className={classes.loadingContainer}>
+        <LoaderContainer>
```

Comment:
this shudnt be updated here, plz undo
this is not part of our code for this pr

Reply:
Done